integrationconnectors/advanced/main.tf

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_project" "default" {
+}
+
+resource "google_secret_manager_secret" "default" {
+  secret_id = "my-example-secrect"
```

Comment:
Please create a secret or as a last resort you can use test skip file to ignore tests for this sample.
```
Error: Error creating Connection: googleapi: Error 400: The request was invalid: Service account 711755423475-compute@developer.gserviceaccount.com does not have permission to access the secret secret_version:"projects/711755423475/secrets/my-example-secrect/versions/1"
```